lib/guard/rack/runner.rb

```diff
@@ -41,7 +41,8 @@ def build_rack_command
       command.push(
         options[:config],
         '--env', options[:environment].to_s,
-        '--port', options[:port].to_s
+        '--port', options[:port].to_s,
+        '--host', options[:host].to_s
```

Comment:
This is taken care of: https://github.com/dblock/guard-rack/pull/17/files#diff-46b6856b2f57a5f4eaf3d1e192a5469cR12